requirements_dev.txt

```diff
@@ -0,0 +1,12 @@
+# Useful development packages.
+# Install them with 'pip install -r requirements_dev.txt'.
+
+memory-profiler
+jedi
+pudb
+pytest-pudb
+pyprof2calltree
+pyinstrument
+# Might fail on some machines, you might have to install from git, with the commented line below:
+vmprof; platform_machine == "x86_64" 
+# git+https://github.com/vmprof/vmprof-python.git
```

Comment:
Just occurred to me: We may want to install that instead of the version from the package index, as that has `vmprofshow flat`, whereas the pypi version doesn't.

Reply:
Done